test/rubygems/test_gem_package.rb

```diff
@@ -1093,6 +1093,11 @@ def test_spec
     assert_equal @spec, package.spec
   end
 
+  def test_gem_attr
+    package = Gem::Package.new(@gem)
+    assert_equal(package.gem, @gem)
```

Comment:
Did you mean `package.gem.path`, here?

Asking due to the nature of the error on Travis CI: https://travis-ci.org/rubygems/rubygems/jobs/557334428#L278

Reply:
I'm confused, why is it expecting a path? It's just a attribute referencing the argument passed to `initialize`.

Reply:
@djberg96 Its because we are calling `self.new` first and then`initialize` https://github.com/rubygems/rubygems/blob/02f8c4b416a62361e721c973a5a23742e0dabbbd/lib/rubygems/package.rb#L141 first and then`initialize`